includes/fields/class-attachment-upload.php

```diff
@@ -85,6 +85,25 @@ public static function init( $meta = [] ) {
 						'type'    => 'checkbox',
 						'_order'  => 120,
 					],
+
+					'multiple'  => [
+						'label'   => esc_html_x( 'Multiple', 'selection', 'hivepress' ),
+						'caption' => esc_html__( 'Allow multiple selection', 'hivepress' ),
+						'type'    => 'checkbox',
+						'_order'  => 130,
+					],
+
+					'max_files' => [
```

Comment:
Is this already gets validated somewhere? I couldn't find the code, or it was an existing one and just the option was not available

Reply:
Yes, just the option was not available. The validation is in Attachment controller for upload attachment action